jumpscale/packages/admin/frontend/api.js

```diff
@@ -167,6 +167,14 @@ const apiClient = {
         headers: {'Content-Type': 'application/json'},
         data: {solution_type: solution_type}
       })
+    },
+    getAll: () => {
+      return axios({
+        url: `/tfgrid_solutions/actors/solutions/list_all_solutions`,
+        method: "post",
+        headers: {'Content-Type': 'application/json'},
+        data: {}
```

Comment:
please remove data, headers and change the method to get

Reply:
removed, thanks :+1: